src/openrct2/world/Footpath.cpp

```diff
@@ -1643,6 +1643,11 @@ void PathElement::SetAddition(uint8_t newAddition)
     Additions = newAddition;
 }
 
+void PathElement::SetAdditionEntryIndex(ObjectEntryIndex entryIndex)
+{
+    Additions = entryIndex + 1;
```

Comment:
Could you add a comment explaining why this needs the plus one here.

Reply:
Done!